app/models/google/calendar/calendar.rb

```diff
@@ -52,7 +52,7 @@ def user_events
       end
       result = client.execute(api_method: service.events.list,
                               parameters: { 'calendarId' => 'priamary',
-                                           'pageToken' => page_token })
+                                            'pageToken' => page_token })
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.